test_simd.py

```diff
@@ -122,11 +122,11 @@ def test_bit_shift() -> None:
 
 def test_to_bool() -> None:
     assert (
-        A(0x_05_00_03_02_01, *s).is_true() ==
+        A(0x_0f_00_03_02_01, *s).is_true() ==
         A(0x_01_00_01_01_01, *s)
     )
     assert (
-        A(0x_05_00_03_02_01, *s).is_false() ==
+        A(0x_0f_00_03_02_01, *s).is_false() ==
```

Comment:
btw I tweaked the test case (0x5 -> 0xf) to give better coverage of the all-ones boundary condition